cirq/sim/density_matrix_simulator_test.py

```diff
@@ -656,3 +656,17 @@ def test_works_on_pauli_string():
     np.testing.assert_allclose(result.reshape(4, 4),
                                np.diag([0, 0, 0, 1]),
                                atol=1e-8)
+
+
+def test_str():
```

Comment:
Rename to `test_density_matrix_trial_result_str()`

Reply:
Done.